paimon-e2e-tests/src/test/java/org/apache/paimon/tests/cdc/MySql57E2eTest.java

```diff
@@ -20,12 +20,21 @@
 
 import org.apache.paimon.flink.action.cdc.mysql.MySqlVersion;
 
+import org.junit.jupiter.api.condition.EnabledIf;
+
 /**
  * E2e tests for {@link org.apache.paimon.flink.action.cdc.mysql.MySqlSyncTableAction} with MySQL
  * 5.7.
  */
+@EnabledIf("runTest")
 public class MySql57E2eTest extends MySqlCdcE2eTestBase {
 
+    private static boolean runTest() {
+        // TODO: modify the following condition after paimon-flink-cdc supports flink 2.0
+        String flinkVersion = System.getProperty("test.flink.main.version");
+        return flinkVersion.compareTo("2.0") < 0;
+    }
+
```

Comment:
Could we move the function to the super class, and sub class override the function when needed?

Reply:
Thanks for the comment. I have added such code to MySqlCdcE2eTestBase. As for FlinkActionsWithKafkaE2eTest, FlinkProceduresE2eTest and LogStoreE2eTest, their runTest() are not reused because the conditions and their parent classes are not exactly the same.